requirements.txt

```diff
@@ -10,3 +10,4 @@ accelerate
 pandas
 pandas-stubs
 lm-eval>=0.4.2
+instructlab
```

Comment:
This doesn't seem right.  Why is it needed here?

Reply:
Yeah we can't have this here - it's a circular dependency

Reply:
++ good feedback, this was something I needed for testing.